vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -42,7 +42,7 @@ The init containers automatically restore a backup included in your application
 The **entando-de-app** is a J2EE application and an instance of the [entando-core](#entando-core). It provides pathways for Entando Components and the server image required by the Entando Operator to manage the deployment. The pom.xml for the application reveals its dependencies.
 
 * GitHub: <https://github.com/entando/entando-de-app/>
```

Comment:
Pretty sure this should be changed to this - https://github.com/entando/app-engine. CC: @pietrangelo @eugeniosant